sed/test/inspec/controls/command_exists.rb

```diff
@@ -0,0 +1,6 @@
+pkg_ident = `grep pkg_ident results/last_build.env | cut -d \= -f2`.strip
```

Comment:
I would rather not "shell out" during an InSpec control like this; but I found that `$pkg_ident` was not available in either the `ENV` environment hash or the `os_env` InSpec resource.

Alternative might be to create an `attributes.yml` file to pass to InSpec.